### PR TITLE
Hw spring project3

### PR DESCRIPTION
**Pull request by Sashk52**

### CODE REVIEW

**Review by ShemsheyMaxim — APPROVED** [review]

File: src/main/java/spring/intro/dto/UserResponceDto.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package spring.intro.dto;
+
+public class UserResponceDto {
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/spring/intro/dto/UserResponceDto.java]:
```suggestion
```

File: src/main/java/spring/intro/service/impl/UserMapper.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package spring.intro.service.impl;
+
+import org.springframework.stereotype.Component;
+import spring.intro.dto.UserResponceDto;
+import spring.intro.model.User;
+import spring.intro.service.UserService;
+
+@Component
+public class UserMapper {
+    private final UserService userService;
```
Comment (dmytro-hryhoruk) [inline src/main/java/spring/intro/service/impl/UserMapper.java]: do you need this?

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/java/spring/intro/dto/UserResponceDto.java (hunk 1 above)
Comment (Sashk52) [reply src/main/java/spring/intro/dto/UserResponceDto.java]: done

File: src/main/java/spring/intro/service/impl/UserMapper.java (hunk 2 above)
Comment (Sashk52) [reply src/main/java/spring/intro/service/impl/UserMapper.java]: No i don't, i was confused with functionalitty of Mapper class. At first, i tried use it. Later changed way of solution and have forgotten to delete it. Done.

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**